src/igl/opengl/RenderCommandEncoder.cpp

```diff
@@ -422,9 +422,7 @@ void RenderCommandEncoder::drawIndexed(size_t indexCount,
   (void)vertexOffset;
   (void)baseInstance;
 
-  IGL_ASSERT_MSG(instanceCount == 1, "Instancing is not implemented");
   IGL_ASSERT_MSG(vertexOffset == 0, "vertexOffset is not implemented");
-  IGL_ASSERT_MSG(baseInstance == 0, "Instancing is not implemented");
```

Comment:
`baseInstance` should not be touched because it is still unsupported in this function.